R/dsl.R

```diff
@@ -784,9 +784,63 @@ list_container <- function(content_list, is_divided = FALSE) {
   list_class <- paste("ui", divided_list, "list")
   div(class = list_class,
       content_list %>% purrr::map(function(x) {
-        if (is.null(x$header) && is.null(x$descriptio))
+        if (is.null(x$header) && is.null(x$description))
           stop("content_list needs to have either header or description.")
         list_element(x$header, x$description, x$icon)
       })
   )
 }
+
+
+#' Accordion UI
+#'
+#' In accordion you may display a list of elements that can be hidden or
+#' shown with one click.
+#'
+#' @param accordion_list lsit with lists with fields: `title` and `content`
+#' @param fluid if accordin is fluid then it takes width of parent div
+#' @param active_title if active title matches `title` from \code{accordion_list}
```

Comment:
`active_title` is not fully working. The element is kinda opened, but not displaying the content. It will be visible after hiding and showing it again:
```
    accordion(
      fluid = FALSE,
      active_title = "Cat",
      list(
        list(
          title = "Dog",
          content = div(p("Dog in some html tags"))
        ),
        list(
          title = "Cat",
          content = "this is cat"
        )
      )
    )
```
![image](https://user-images.githubusercontent.com/19154385/91533807-cf9ad200-e910-11ea-8b42-d14176e54a2e.png)